core/input/input.h

```diff
@@ -219,6 +219,14 @@ class Input : public Object {
 
 	Vector<JoyDeviceMapping> map_db;
 
+	struct JoyAxisButtonMapping {
+		bool replace_axis_as_button = false;
+		float deadzone = 0.2;
+		float previous_value = 0.0;
+	};
+
+	LocalVector<LocalVector<JoyAxisButtonMapping>> axis_to_button_mapper;
```

Comment:
Why not `HashMap<int, LocalVector>` ?

Reply:
I was under the impression, that `LocalVector` was more appropriate than `HashMap`, because I can access the content by integer indices, so there is no need for creating the hash values.